src/hash-utils.ts

```diff
@@ -53,7 +53,9 @@ export function objectToJsonString(keys: string[], values: string[]): string {
   let result = new Array<string>(keys.length);
 
   for (let i = 0; i < keys.length; ++i) {
-    result[i] = `${addQuotes(keys[i])}:${addQuotes(values[i])}`;
+    result[i] = `${addQuotes(keys[i])}:${addQuotes(
+      values[i].replaceAll('"', '\\"'),
```

Comment:
Description says:
> I will replace quotes with triple backslashes + quote for every occurrence in the key strings.

But this code replaces with _double_ backslashes + quote.

Please synchronize the code and the description. I'm not sure which one is correct. 

# Reference
A quick test shows that running `JSON.stringify()` on the key string yields triple backslashes.

```js
(async () => {
    const key = '05{"ciphertext":"lD1tyGbUIlYx6ll1DpBxFvz4QRF5WaJx7uksMgK+rA82AG6Gw0FujU4yH/tulrZQxkwMnTJ+vw5UVBhLUTgYABTIZth9cFAy6uP7u/URqVgtowe+JJmZTIeVBcVItOQrBCKjV7jlJWRuL5FtAdHEZhGJxnz1Yce+pngu1ztQAg==","dataToEncryptHash":"98032a673081e760a026588dfca7dc5f23f00121a48d8e4d225851e6e63fbdca"}';

    console.log(JSON.stringify(JSON.stringify(key)));
  })();
```

Output
```
"\"05{\\\"ciphertext\\\":\\\"lD1tyGbUIlYx6ll1DpBxFvz4QRF5WaJx7uksMgK+rA82AG6Gw0FujU4yH/tulrZQxkwMnTJ+vw5UVBhLUTgYABTIZth9cFAy6uP7u/URqVgtowe+JJmZTIeVBcVItOQrBCKjV7jlJWRuL5FtAdHEZhGJxnz1Yce+pngu1ztQAg==\\\",\\\"dataToEncryptHash\\\":\\\"98032a673081e760a026588dfca7dc5f23f00121a48d8e4d225851e6e63fbdca\\\"}\""
```